src/widgetastic_patternfly/__init__.py

```diff
@@ -132,6 +132,21 @@ def title(self):
         return self.browser.get_attribute('title', self)
 
 
+class SaveButton(Button):
```

Comment:
I wouldn't create a separate class for that. Just enough to change `disabled` property in the parent class:
```python
@property
def disabled(self):
    return self.browser.get_attribute('disabled', self) == 'disabled' or 'disabled' in self.browser.classes(self)
```

Reply:
I made a new class because I think some of the button will work correctly with disabled property of Button and some will use SaveButton's property, like I am using it in one of my test case.

Reply:
I'd agree with @quarckster here, we don't need a separate class for the disabled property, we'd be much better served by an improved disabled property in the existing button class.

Reply:
Okay @quarckster @mshriver will implement the suggested changes. I didn't implement it on the first place because I thought that there will some buttons which have disabled property working properly.